src/Tgstation.Server.Host/Setup/SetupWizard.cs

```diff
@@ -841,8 +842,11 @@ async Task<Uri> ParseAddress(string question)
 				{ SwarmConfiguration.Section, swarmConfiguration },
 			};
 
-			var json = JsonConvert.SerializeObject(map, Formatting.Indented);
-			var configBytes = Encoding.UTF8.GetBytes(json);
```

Comment:
Can you check and see what is still `using Newtonsoft.Json;` in this file? It should be nothing at this point.